src/components/Content/TaskHierarchy/TaskHierarchy.js

```diff
@@ -96,17 +96,30 @@ class TaskHierarchy extends Component {
     )
   }
 
+  produceOptions = () => {
+    const pursuanceArr = Object.values(this.props.pursuances);
```

Comment:
@thevaleriemack Want to sort the pursuances by ID here?

Reply:
done!